egs/wsj/s5/utils/data/subsegment_data_dir.sh

```diff
@@ -50,11 +51,23 @@ export LC_ALL=C
 
 srcdir=$1
 subsegments=$2
-new_text=$3
-dir=$4
 
+no_text=true
```

Comment:
In general, having negation as part of the name of a boolean variable is a bad policy as it makes code hard to understand.  But in this case I think it would be easier to just make the "new_text" variable empty and forgo the separate boolean variable.  [of course, if num-args=4, check that the variable passed in is nonempty.]